KaitaiStream.js

```diff
@@ -562,14 +562,18 @@ KaitaiStream.processRotateLeft = function(data, amount, groupSize) {
 }
 
 KaitaiStream.processZlib = function(buf) {
-  if (typeof KaitaiStream.zlib === 'undefined')
-    KaitaiStream.zlib = require('zlib');
-  if (buf instanceof Uint8Array) {
-    var b = new Buffer(buf.buffer);
+  var r;
+  if (typeof require === 'undefined')  {
+    // no require() -> Assume we're running as a webworker in the browser
+    importScripts("../../lib/_npm/pako/pako_inflate.min.js");
```

Comment:
This seems to refer some paths inside web IDE, and runtime definitely must not include any hard codes tailored specifically for web IDE. Lots of people use JS runtime in a browser outside of web IDE, and we shouldn't force their projects to use some specific paths or whatever.

Reply:
made configurable